### PR TITLE
add GEPRCF745_BT_HD board.

### PR DESCRIPTION
**Pull request by Linjieqiang**

### CODE REVIEW

File: configs/default/GEPR-GEPRCF745_BT_HD.config [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+serial 0 1 115200 57600 0 115200
+serial 1 64 115200 57600 0 115200
+serial 2 1 115200 57600 0 115200
+serial 7 1024 115200 57600 0 115200
+
+# aux
+aux 0 0 0 900 1300 0 0
+aux 1 1 1 1700 2100 0 0
+aux 2 28 1 900 1300 0 0
+aux 3 40 4 900 1300 0 0
```
Comment (DusKing1) [inline configs/default/GEPR-GEPRCF745_BT_HD.config:127]: what are these for?

File: configs/default/GEPR-GEPRCF745_BT_HD.config [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+# master
+set gyro_to_use = BOTH
+set mag_i2c_device = 1
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set serialrx_provider = SBUS
+set blackbox_sample_rate = 1/1
+set blackbox_device = SDCARD
+set dshot_bidir = ON
+set motor_pwm_protocol = DSHOT600
```
Comment (DusKing1) [inline configs/default/GEPR-GEPRCF745_BT_HD.config:138]: Should be removed

**Review by DusKing1 — CHANGES_REQUESTED** [review]

File: configs/default/GEPR-GEPRCF745_BT_HD.config (hunk 1 above)
Comment (Linjieqiang) [reply configs/default/GEPR-GEPRCF745_BT_HD.config:127]: Hello @DusKing1 . These aux settings are mainly used to configure the use of Bluetooth and digital switches for user priority. So it should be kept here.

File: configs/default/GEPR-GEPRCF745_BT_HD.config (hunk 2 above)
Comment (Linjieqiang) [reply configs/default/GEPR-GEPRCF745_BT_HD.config]: Fixed.

**Review by haslinghuis — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**